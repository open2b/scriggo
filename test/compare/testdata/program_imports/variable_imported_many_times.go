// skip : https://github.com/open2b/scriggo/issues/492

// rundir