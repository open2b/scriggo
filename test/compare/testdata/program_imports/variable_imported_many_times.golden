a: c.C is 42 (before increment)
a: c.C is 52 (after increment)
b: c.C is 52
